models/pull.go

```diff
@@ -69,15 +69,16 @@ type PullRequest struct {
 	Issue   *Issue `xorm:"-"`
 	Index   int64
 
-	HeadRepoID      int64                  `xorm:"INDEX"`
-	HeadRepo        *repo_model.Repository `xorm:"-"`
-	BaseRepoID      int64                  `xorm:"INDEX"`
-	BaseRepo        *repo_model.Repository `xorm:"-"`
-	HeadBranch      string
-	HeadCommitID    string `xorm:"-"`
-	BaseBranch      string
-	ProtectedBranch *ProtectedBranch `xorm:"-"`
-	MergeBase       string           `xorm:"VARCHAR(40)"`
+	HeadRepoID                int64                  `xorm:"INDEX"`
+	HeadRepo                  *repo_model.Repository `xorm:"-"`
+	BaseRepoID                int64                  `xorm:"INDEX"`
+	BaseRepo                  *repo_model.Repository `xorm:"-"`
+	HeadBranch                string
+	HeadCommitID              string `xorm:"-"`
+	BaseBranch                string
+	ProtectedBranch           *ProtectedBranch `xorm:"-"`
+	MergeBase                 string           `xorm:"VARCHAR(40)"`
+	AllowEditsFromMaintainers bool
```

Comment:
Same here

Reply:
ps: the name is so long, maybe `AllowMaintainerEdit` is enough. ps ps: both are fine to me.  :)

Reply:
See https://github.com/go-gitea/gitea/pull/18002/commits/4d5b07387ba71ccc2b16c63ec55c430d6cd0f0a4